src/wasm/kotlin/src/arcs/WasmJsInterop.kt

```diff
@@ -186,7 +190,7 @@ external fun collectionClear(particlePtr: WasmAddress, handlePtr: WasmAddress)
 external fun render(particlePtr: WasmAddress, slotNamePtr: WasmString, templatePtr: WasmString, modelPtr: WasmString)
 
 @SymbolName("_onRenderOutput")
-external fun onRenderOutput(particlePtr: WasmAddress, templatePtr: WasmString, modelPtr: WasmString)
+external fun onRenderOutput(particlePtr: WasmAddress, templatePtr: WasmNullableString, modelPtr: WasmNullableString)
```

Comment:
KDoc would be great here too...

Reply:
I'm going to add KDocs to all these sdk files in chunks over a few PRs (just to break up the large task). 

This file will be next or towards the top of the list.